packages/manager/apps/dedicated/client/app/ip/ip-expand-ipv6.service.js

```diff
@@ -0,0 +1,55 @@
+// [JS] Expand Abbreviated IPv6 Addresses
+// by Christopher Miller
+// http://forrst.com/posts/JS_Expand_Abbreviated_IPv6_Addresses-1OR
+// Modified to work with embedded IPv4 addresses
+export default class {
```

Comment:
- conversion to a `class`
- disabling need for `this` by `ESlint`, it would have too much impact
- nothing else has been done